changelog.md

```diff
@@ -190,6 +190,31 @@ let
 
 - Added support for casting between integers of same bitsize in VM (compile time and nimscript).
   This allow to among other things to reinterpret signed integers as unsigned.
+
 - Pragmas now support call syntax, for example: ``{.exportc"myname".}`` and ``{.exportc("myname").}``
 - Custom pragmas are now supported using pragma ``pragma``, please see language manual for details
 
+- Added True Color support for some terminals
+  Example:
+```nim
+import colors, terminal
+
+const Nim = "Efficient and expressive programming."
+
+var
+  fg = colYellow
+  bg = colBlue
+  int = 1.0
```

Comment:
a variable called 'int' that is of type float? Come on. ;-)

Reply:
intensity :-)

Reply:
This needs to be changed.